### ISSUE TITLE
Remirror is broken

### ISSUE THREAD

**Issue by faisalsayed10** [remirror/remirror#2108]
### Summary

There's some random error that says `Cannot read properties of undefined (reading "messages")`. And it's everywhere. I was panicking whether there's something wrong within my app but then I went to remirror's storybook page and literally every story there is broken with the same error message.

```
TypeError: Cannot read properties of undefined (reading 'messages')
    at _ (https://remirror.vercel.app/assets/remirror-react-core-f93566c3.js:82:47871)
    at description (https://remirror.vercel.app/assets/remirror-extension-tables-010b0786.js:1:36671)
    at Fi (https://remirror.vercel.app/assets/remirror-react-components-02d92985.js:99:47079)
    at https://remirror.vercel.app/assets/remirror-react-components-02d92985.js:99:48042
    at Object.useMemo (https://remirror.vercel.app/assets/index-9c09ad76.js:22:25729)
    at u.useMemo (https://remirror.vercel.app/assets/index-ebeaab24.js:9:6060)
    at cp (https://remirror.vercel.app/assets/remirror-react-components-02d92985.js:99:48016)
    at me (https://remirror.vercel.app/assets/remirror-react-components-02d92985.js:99:49585)
    at eu (https://remirror.vercel.app/assets/index-9c09ad76.js:22:19476)
    at ka (https://remirror.vercel.app/assets/index-9c09ad76.js:24:43691)
```

### Steps to reproduce

go to https://remirror.vercel.app/?path=/story/editors-wysiwyg--basic, it's broken.

### Screenshot(s)

<img width="1440" alt="image" src="https://github.com/remirror/remirror/assets/36627266/39d794d9-6136-4f54-b05a-dfd2476ae707">

**Comment by faisalsayed10** [comment]
Damn! Thank you for such a quick fix

**Comment by ocavue** [comment]
This bug should be fixed in `@remirror/react` version 2.0.32. Thank you for reporting this!
